src/main/kotlin/org/rust/cargo/runconfig/command/CargoExecutableRunLineMarkerContributor.kt

```diff
@@ -0,0 +1,27 @@
+package org.rust.cargo.runconfig.command
+
+import com.intellij.execution.lineMarker.ExecutorAction
+import com.intellij.execution.lineMarker.RunLineMarkerContributor
+import com.intellij.icons.AllIcons
+import com.intellij.psi.PsiElement
+import com.intellij.util.Function
+import org.rust.lang.core.psi.RsElementTypes.IDENTIFIER
+import org.rust.lang.core.psi.RsFunction
+import org.rust.lang.core.psi.util.elementType
+
+class CargoExecutableRunLineMarkerContributor : RunLineMarkerContributor() {
+    override fun getInfo(element: PsiElement): Info? {
+        if (element.elementType != IDENTIFIER) return null
+        val fn = element.parent as? RsFunction ?: return null
+        if (!CargoExecutableRunConfigurationProducer.isMainFunction(fn)) return null
+
+        val actions = ExecutorAction.getActions(0)
```

Comment:
BTW: Is it possible to filter available executors? (overally, not here, though filtering by `toString()` or by using reflection to extract private fields isn't good idea either)

User gets spammed with `Debug` and `Run with Coverage` actions which are unavailable as for now.